build/yamls/base/agent.yml

```diff
@@ -128,7 +128,7 @@ spec:
             subPath: cni
           # We need to mount both the /proc directory and the /var/run/netns directory so that
           # antrea-agent can open the network namespace path when setting up Pod
-          # networking. Different container runtimes may use /proc or /var/run/netns when invoking
+          # controlplane. Different container runtimes may use /proc or /var/run/netns when invoking
```

Comment:
is this change expected?

Reply:
I used Goland to do the refactoring. Observed it introduced some unexpected changes too. Let me check.